[MB-8641] Display counselor and customer remarks on Move Task Order view

hannaliebl opened the pull request:
## Description

Display counselor and customer remarks on Move Task Order view.

## Reviewer Notes

I tested this locally with long form text data and it worked find without any white space styles, as best I could tell, but maybe taking a look at that.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make server_run
make client_run
```
Login as TOO, and on MTO page, if the shipment has customer or counselor remarks, they now show up there.

You can also look at the Shipment Details component in Storybook.

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely) have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-8641) for this change



## Screenshots

![Screen Shot 2021-07-13 at 3 38 56 PM](https://user-images.githubusercontent.com/3903208/125514647-28e388ae-a8a4-4de4-9a64-4ac28462e8a2.png)

robot-mymove commented:
<!--
  0 failure: 
  0 warning: 
  1 messages
  
  DangerID: danger-id-Danger;
-->



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>:link: <a href="https://dp3.atlassian.net/browse/MB-8641">MB-8641</a></td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 342a3edd6d308284a000787fff3a6b746bb3ade5
</p>

duncan-truss commented:
@hannaliebl This is looking good.  Two styling adjustments I can think of to incorporate are:
- `white-space: prewrap` for preserving paragraph formatting
- The bottom margin between the last remarks container and the service items header should be 4 units = 32px from the Zeplin mocks I looked at.  Right now it looks to be 24px + 22px when I looked in the inspector.

hannaliebl commented:
I added the pre-wrap style. I am wondering about Duncan's other comment: should I change the H3 margin below the tables? Do something else? I'm a little leery to change a bunch of stuff because a lot of this is unrelated to what I added. Thoughts, @transcom/truss-design?

jenniferlynparsons commented on src/components/Office/ShipmentDetails/ShipmentDetailsMain.jsx:
```diff
@@ -34,6 +35,8 @@ const ShipmentDetailsMain = ({ className, shipment, dutyStationAddresses, handle
         handleDivertShipment={handleDivertShipment}
       />
       <ShipmentWeightDetails estimatedWeight={primeEstimatedWeight} actualWeight={primeActualWeight} />
+      {shipment.counselorRemarks && <ShipmentRemarks title="Counselor remarks" remarks={shipment.counselorRemarks} />}
```
Since we're already deconstructing other values from the shipment earlier (line 13) we should keep that pattern consistent for `counselorRemarks` and `customerRemarks`.

jenniferlynparsons reviewed (changes requested):
Looks good in storybook. Just a small consistency change and this is good to go.

donaldthai commented:
> I added the pre-wrap style. I am wondering about Duncan's other comment: should I change the H3 margin below the tables? Do something else? I'm a little leery to change a bunch of stuff because a lot of this is unrelated to what I added. Thoughts, @transcom/truss-design?

I think you can just decrease the margin for that last remarks container to get to 4 units. Zappo: 
![image](https://user-images.githubusercontent.com/1522549/125859446-fbc17f4f-fbdd-4182-9f37-b7028a97581a.png)

jenniferlynparsons reviewed (approved):
Good to go!
